azure-pipelines.yml

```diff
@@ -600,6 +600,39 @@ jobs:
         inlineScript: |
           Get-ChildItem .vault-config/*.yaml |% { dotnet secret-manager synchronize $_}
 
+  # Ubuntu 1804 x64 ILLink Benchmarks
```

Comment:
I realized that you will also want to add this to the public correctness run section as well. You will want your stuff run to make sure that the tests don't get broken.